src/panels/lovelace/hui-root.ts

```diff
@@ -282,11 +291,16 @@ class HUIRoot extends LitElement {
       haStyle,
       css`
         :host {
-          tabindex: -1,
           --dark-color: #455a64;
           --text-dark-color: #fff;
         }
 
+        .disable-text-select {
```

Comment:
Use `:host(.disable-text-select)` because the class is not inside this shadowroot but on the host.

Reply:
I was reading up on that in bed last night, thanks!